docs/src/administration/cli/_index.md

```diff
@@ -189,3 +189,28 @@ This isn't available by default on macOS, but can be installed via `brew`.
 Check your home directory and ensure that the file `~/.platformsh/autocompletion.sh` is being included by your shell.
 
 If you experience issues, run `platform self:install` to attempt a reinstall of this utility.
+
+### Run commands on your container
+
+You can use the Platform.sh CLI to run commands on your container.
+You can use any command you've added in dependencies or the build hook.
```

Comment:
⛏️  A link to the dependencies and the build hook might be nice.
- https://cli-ssh-commands-aeihkoy-652soceglkw4u.eu-3.platformsh.site/create-apps/app-reference.html#dependencies
- https://cli-ssh-commands-aeihkoy-652soceglkw4u.eu-3.platformsh.site/create-apps/hooks.html#compile-extension